src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+/**
+ * Deals with making sense of the user command
+ *
+ * @author sibinhho99-nus
+ */
+public class Parser {
+    /**
+     * Returns index to delete from delete command
+     */
+    public static int getDeleteIndex(String command) {
+        // Extract name from todo command, eg. todo borrow book
+        return Integer.parseInt(command.split(" ")[1]) - 1;
+    }
+
+    /**
+     * Returns task name to find from find command
+     */
+    public static String getTaskToFindName(String command) {
+        return command.substring(5);
+    }
+
+    /**
+     * Returns the type of the command (bye delete etc)
+     */
+    public String getType(String command) {
+        return command.split(" ")[0];
+    }
+
+    /**
+     * Returns event Task object from command
+     */
+    public Task parseEvent(String command) {
+        // Extracts name and time to be done by from event command,
+        // eg. event project meeting /at Mon 2-4pm
+        String temp = command.substring(6);
```

Comment:
Doesn't hardcoding a '6' (and '9' later on in parseDeadlines) make the program quite hard to understand? Perhaps try using `Pattern` and `Matcher` from `java.util.regex` to extract out the date?